src/client/scss/icons.scss

```diff
@@ -66,12 +66,14 @@
 }
 .icon-functional::before {
   content: '\ea0b';
+  visibility: hidden;
```

Comment:
I'm not sure it's any less of a "hack", but setting `content: ' ';` seems to work for me as well. Either way is fine.